### PR TITLE
ci: enable Packit rpm build test

### PR DESCRIPTION
**Pull request by jamacku**
Documentation: https://packit.dev/docs

### CODE REVIEW

File: .packit.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+specfile_path: strace.spec
+upstream_package_name: strace
+downstream_package_name: strace
+
+# package list from ci/install-dependencies.sh + packages needed for make dist and rpm build
+srpm_build_deps:
+  - autoconf
+  - automake
+  - autogen
```
[the remark this answers is not in the document]
Comment (jamacku) [reply .packit.yml:12]: I'm not sure. Let's remove it and see if it works without `autogen`.

File: .packit.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  post-upstream-clone:
+    - ./bootstrap
+    - ./configure --disable-dependency-tracking --enable-Werror --enable-maintainer-mode
+  create-archive:
+    - make dist-gzip
+    - bash -c "ls ./strace-*.tar.gz"
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
```
Comment (ldv-alt) [inline .packit.yml:26]: Is it possible to trigger this also in case of a push?

Comment (jamacku) [reply .packit.yml:26]: Yes it is, I'll update the configuration

File: .packit.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Docs: https://packit.dev/docs/
```
Comment (ldv-alt) [inline .packit.yml:1]: Strangely enough, https://packit.dev/docs/ says `Page Not Found`, while https://packit.dev/docs works.

File: .packit.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    targets:
+      - fedora-rawhide-aarch64
+      - fedora-rawhide-i386
+      - fedora-rawhide-ppc64le
+      - fedora-rawhide-s390x
+      - fedora-rawhide-x86_64
+
+  - <<: *copr
+    trigger: commit
+    branch: main
```
Comment (ldv-alt) [inline .packit.yml:37]: We don't have `main` branch yet.  Does the syntax accepts patterns or lists?

File: .packit.yml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  - libtool
+
+actions:
+  # generate spec file from input file
+  post-upstream-clone:
+    - ./bootstrap
+    - ./configure --disable-dependency-tracking --enable-Werror --enable-maintainer-mode
+  create-archive:
+    - make dist-gzip
+    - bash -c "ls ./strace-*.tar.gz"
```
Comment (ldv-alt) [inline .packit.yml:22]: I suppose we can use `sh` instead of `bash` here.

File: .packit.yml (hunk 3 above)
Comment (jamacku) [reply .packit.yml]: Fixed

**Comment by ldv-alt** [comment]
Would it make sense to merge this PR with fedora-rawhide-aarch64 target omitted while we're waiting for #305?

**Comment by jamacku** [comment]
Sure, I will temporarily comment out the aarch64 target.

**Review by ldv-alt — APPROVED** [review]

**Pull request merged**